CRM/Core/BAO/CustomGroup.php

```diff
@@ -663,6 +632,35 @@ public static function &getTree(
   }
 
   /**
+   * Clean and validate the filter before it is used in a db query.
+   *
+   * @param string $entityType
+   * @param string $subType
+   *
+   * @return string
+   * @throws \CRM_Core_Exception
+   * @throws \CiviCRM_API3_Exception
+   */
+  protected static function validateSubTypeByEntity($entityType, $subType) {
+    $subType = trim($subType, CRM_Core_DAO::VALUE_SEPARATOR);
```

Comment:
Not quite understanding why we need this (was a comma-separated list above), but trim() cannot hurt anyway.